frame/system/src/lib.rs

```diff
@@ -424,7 +424,7 @@ decl_storage! {
 		Digest get(fn digest): DigestOf<T>;
 
 		/// Events deposited for the current block.
-		Events get(fn events): Vec<EventRecord<T::Event, T::Hash>>;
+		pub Events get(fn events): Vec<EventRecord<T::Event, T::Hash>>;
```

Comment:
```suggestion
		Events get(fn events): Vec<EventRecord<T::Event, T::Hash>>;
```

`system::Module::events()` is what you want